book/environment.yml

```diff
@@ -0,0 +1,10 @@
+name: climaterisk
+channels:
+  - conda-forge
+dependencies:
+  - python==3.9
```

Comment:
Is pinning to Python 3.9 required? It will reach end of life by 2025-10 and though that's technical beyond the end of our grant (2025-05-30) it seems like an odd choice to start with.

Reply:
That's a good point. In the past I remember coming across situations where conda resolved the environment in a manner that resulted in a fairly old version of Python being installed into the environment. This is meant to avoid that situation.

I'll check and update the environment file.